src/Exceptions/InvalidReplacer.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Spatie\ResponseCache\Exceptions;
+
+use Exception;
+
+class InvalidReplacer extends Exception
```

Comment:
Maybe I'm looking over it, but is this one actually being thrown somewhere?

Reply:
From old implementation, deleted